src/models/eventmodel.cpp

```diff
@@ -25,7 +25,9 @@ enum class Tag : quint8
     Overview = 2,
     Cpus = 3,
     Processes = 4,
-    Threads = 5
+    Threads = 5,
+    Tracepoints = 6,
+    LAST_TAG,
```

Comment:
I dislike how that pollutes all switch statements. please instead introduce just below the Tag enum:

```
constexpr auto LAST_TAG = Tag::Tracepoints;
```